atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/queue/WriteInfoPartitionerTest.java

```diff
@@ -23,6 +23,7 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
+import static com.palantir.atlasdb.sweep.queue.SweepQueueTablesTest.metadataBytes;
```

Comment:
Static method import. Maybe this is personal preference? Not sure if we have specific guidelines about it.

Reply:
Checkstyle doesn't complain, and we routinely do it for `Mockito`. I'm fine either way, so if it is too jarring, will remove :)

Reply:
OK, ignore me then. Your point about readability in the other comment is valid, too.